src/Microsoft.Framework.Runtime.Interfaces/IApplicationShutdown.cs

```diff
@@ -2,10 +2,19 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// Exposes methods that allow control over the application lifetime
+    /// </summary>
     public interface IApplicationShutdown
     {
+        /// <summary>
+        /// Terminates the current application.
```

Comment:
Let's update the docs to reflect that then :)

> Requests that the current application be terminated as soon as possible.

Remarks explaining why this is a "request" and who actually terminates the app would be good too.